src/translations/en.json

```diff
@@ -487,7 +487,7 @@
             "rpi_gpio": "[%key:ui::panel::config::server_control::section::reloading::rpi_gpio%]"
           },
           "server_control": {
-            "perform_action": "{action} Server",
+            "perform_action": "{action} server",
```

Comment:
You can remove this. I'll account for it in the navigation commands PR (which adds `toTitleCase()`):
https://github.com/home-assistant/frontend/pull/7380/files#diff-41875554b4bb188dedf0f09df77aa9455d8070fd0afe26876451496db2b485d7R334

Due to the timing of that PR, it currently only title-cases the reload and navigation commands. I'll add it to the server control commands as well.